sympy/polys/polyutils.py

```diff
@@ -211,7 +211,7 @@ def _parallel_dict_from_expr_if_gens(exprs, opt):
                         if not factor.free_symbols.intersection(opt.gens):
                             coeff.append(factor)
                         else:
-                            raise PolynomialError("%s contains an element of the generators set" % factor)
+                            raise PolynomialError("%s contains an element of the set of generators" % factor)
```

Comment:
Can you also break down this line to <= 79 characters?

Reply:
Done. @gxyd